server/utils/mail.js

```diff
@@ -6,7 +6,7 @@ import nodemailer from 'nodemailer';
 import mg from 'nodemailer-mailgun-transport';
 
 const auth = {
-  api_key: process.env.MAILGUN_KEY,
+  api_key: process.env.MAILGUN_KEY || process.env.MAILGUN_API_KEY,
```

Comment:
why these two different environment variable names?

Reply:
Again, same thing here- it's just the way that Mailgun has chosen to name their env variables when you provision their service...